src/irmin-pack/conf.mli

```diff
@@ -23,7 +23,13 @@ type inode_child_order =
 
 module type S = sig
   val entries : int
+  (** The branching factor of the B-tree used for storing inodes. 32 is a good
+      choice for general applications. *)
```

Comment:
I don't think inode trees qualify as B-trees. Wikipedia sais `In B-trees, internal (non-leaf) nodes can have a variable number of child nodes`. In inode trees all child nodes (i.e. steps pointing to `nodes` or `contents`) live in the leaves of the graph.

It's however true that `entries` is the branching factor of inode trees.